schema/me/notifications.js

```diff
@@ -46,6 +46,10 @@ const NotificationsFeedItemType = new GraphQLObjectType({
         },
       }),
     },
+    image: {
+      type: GraphQLString,
+      resolve: ({ object }) => object.artist.image_url,
```

Comment:
Sounds good. I was also thinking of changing the name of the field to `artist_image` or `artist_avatar_image` or something like that to avoid confusion